lib/Request.js

```diff
@@ -48,9 +49,16 @@ Request.prototype.setUrl = function(url) {
         return value;
     });
 
+    //Remove props for query parameters
+    this.deleteQuery('props');
+
     this._etag = null;
 };
 
+Request.prototype.getValidationErrors = function() {
+    return this._validationErrors;
```

Comment:
Since this method is public, should its return value be lazy-loaded? Is there a chance that this can be called before `this._validationErrors` has been populated?